internal/namespaces/account/custom.go

```diff
@@ -0,0 +1,128 @@
+package account
+
+import (
+	"context"
+	"io/ioutil"
+	"os"
+	"path"
+	"reflect"
+	"strings"
+
+	"github.com/scaleway/scaleway-cli/internal/args"
+	"github.com/scaleway/scaleway-cli/internal/core"
+	"github.com/scaleway/scaleway-cli/internal/interactive"
+	account "github.com/scaleway/scaleway-sdk-go/api/account/v2alpha1"
+	"github.com/scaleway/scaleway-sdk-go/scw"
+)
+
+func GetCommands() *core.Commands {
+	return core.NewCommands(
+		sshKeyCommand(),
+		initCommand(),
+	)
+}
+
+func sshKeyCommand() *core.Command {
+	return &core.Command{
+		Short:     `Manage SSH key`,
+		Long:      `Manage SSH key.`,
+		Namespace: "account",
+		Resource:  "ssh-key",
+		NoClient:  true,
+		ArgsType:  reflect.TypeOf(args.RawArgs{}),
+	}
+}
+
+type InitArgs struct {
+	WithSSHKey *bool
+}
+
+func initCommand() *core.Command {
+	return &core.Command{
+		Short:     `Initiliaze SSH key`,
+		Long:      `Initiliaze SSH key.`,
+		Namespace: "account",
+		Resource:  "ssh-key",
+		Verb:      "init",
+		ArgsType:  reflect.TypeOf(InitArgs{}),
+		ArgSpecs: core.ArgSpecs{
+			{
+				Name:  "with-ssh-key",
+				Short: "Whether the ssh key for managing instances should be uploaded automatically",
+			},
+		},
+		Run: InitRun,
+	}
+}
+
+func InitRun(ctx context.Context, argsI interface{}) (i interface{}, e error) {
+	args := argsI.(*InitArgs)
+
+	// Explanation
+	_, _ = interactive.Println("An SSH key is required if you want to connect to a server. More info at https://www.scaleway.com/en/docs/configure-new-ssh-key/")
+
+	// Get default SSH key locally
+	relativePath := path.Join(".ssh", "id_rsa.pub")
+	filename := path.Join(core.ExtractEnv(ctx, "HOME"), relativePath)
+	shortenedFilename := "~/" + relativePath
+	localSSHKeyContent, err := ioutil.ReadFile(filename)
+
+	addKeyInstructions := `scw account ssh-key add name=my-key key="($)(cat path/to/my/key.pub)"`
```

Comment:
because of `($)`  this command will work neither for bash not fish user without editing.
Either keep the bash syntax or create a helper to handle both shell